ceph.spec.in

```diff
@@ -2135,6 +2157,16 @@ fi
 %{python3_sitelib}/ceph_daemon.py
 %{python3_sitelib}/__pycache__/ceph_daemon.cpython*.py*
 
+%if 0%{with python2}
+%files -n python-ceph-common
+%{python_sitelib}/ceph
+%{python_sitelib}/ceph-*.egg-info
+%endif
+
+%files -n python%{python3_pkgversion}-ceph-common
+%{python3_sitelib}/ceph
+%{python3_sitelib}/ceph-*.egg-info
```

Comment:
> Same issue in the RPM file. And idea would be to simply disable the creation of the python 3 package if `0%{with python2}`

i think we build python3 packages all the time in `ceph.spec.in`. so the fix is do not install python-ceph-common using python2 if python2 is not enabled.

Reply:
FWIW I was never able to get both Python3 and Python2 built at the same time, it always picked one. I am not sure what it will take to produce both

Reply:
okay, lemme give it a stab tomorrow.

Reply:
Doesn't the MGR have an embedded Python subinterpreter? And it can be either 2 or 3, but not both?

Reply:
@smithfarm i am on the same page with you. see https://github.com/ceph/ceph/pull/28915#discussion_r306895859

the reason packaging py2 *and* py3 python-ceph-common *might* be useful is just based on the comment of

> This package is supposed to contain common Python code usable by all modules, daemons, cython bindings and tools.

but @sebastian-philipp , it would be ideal if you could help clarify how this python-ceph-common will be used by, for instance, cython bindings?